jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerModule.java

```diff
@@ -0,0 +1,30 @@
+package org.secondbase.logging;
+
+import org.secondbase.core.SecondBase;
+import org.secondbase.core.config.SecondBaseModule;
+import org.secondbase.flags.Flag;
+
+public class JsonLoggerModule implements SecondBaseModule {
+
+    @Flag(name = "environment",
+            description = "The environment the service runs in (test|staging|production).")
+    private static String environment = "test";
+
+    @Flag(name = "service", description = "The name of the service (testservice).")
+    private static String service = "testservice";
+
+    @Flag(name = "datacenter",
+            description = "The datacenter the service runs in (eu-west-1|ap-southeast-1).")
+    private static String datacenter = "eu-west-1";
```

Comment:
I removed all defaults. This also means I can't abuse the loadOpts method any longer. One solution I see is a second loop over the modules and calling their init methods in that loop (after the parameters are parsed).

I'm not sure I'm convinced though. Right now I am reading the values in the main method only. Just to make this work for now.